### PR TITLE
ncm-metaconfig: add service beats and logstash plugin

### PR DESCRIPTION
**Pull request by stdweird**
Support for [beats](https://www.elastic.co/products/beats) (and beats logstash input)

### CODE REVIEW

File: ncm-metaconfig/src/main/metaconfig/beats/pan/schema.pan [hunk 1]
```diff
@@ -0,0 +1,159 @@
+declaration template metaconfig/beats/schema;
+
+include 'pan/types';
+
```
Comment (kwaegema) [inline ncm-metaconfig/src/main/metaconfig/beats/pan/schema.pan]: documentation strings for the types in this schema?

**Comment by jrha** [comment]
Please improve the description, I have provided a template :stuck_out_tongue:

**Comment by stdweird** [comment]
@jrha is that enough?

**Comment by jrha** [comment]
Lovely.

**Comment by jrha** [comment]
@alahiff might like this one.

**Comment by alahiff** [comment]
Yes, this will definitely be useful.

File: ncm-metaconfig/src/main/metaconfig/beats/pan/schema.pan (hunk 1 above)
Comment (jrha) [inline ncm-metaconfig/src/main/metaconfig/beats/pan/schema.pan]: @stdweird could you respond?

Comment (stdweird) [inline ncm-metaconfig/src/main/metaconfig/beats/pan/schema.pan]: ok, i'll add them

**Comment by stdweird** [comment]
@jrha @kwaegema is that sufficient for now?

**Pull request merged**